### ISSUE TITLE
[Woot] Impossible to capture Chaos Guardian in Ender Shard

### ISSUE THREAD

**Issue by UncertifiedDevice** [AllTheMods/ATM-3#418]
Just as the title says, when you hit the chaos guardian with an Ender Shard some text pops up in chat saying that you cannot capture this mob. 

I've played around a bit and found that doesn't matter if the chaos guardian is being protected by crystals or if it has just 1 hp, it simply won't capture. It doesn't look like the chaos guardian is blacklisted in config but it still doesn't work.

I dug around a bit and found this [forum post](https://forum.feed-the-beast.com/threads/chaos-guardian-in-woot-factory.221832/) on ftb forums. In this post Ipsis says that 

> "I will be blacklisting the Chaos Guardian along with a bunch of other mobs that don't play nice with the loot learning mechanics."

 I don't know if this is hardcoded into the mod so I'll create an issue on the woot github referencing this one.

**Comment by Ipsis** [comment]
Since ATM has intentionally configured Woot to allow capturing the Chaos Guardian, is this something that people want supported? Apart from its custom dying mechanics, I had always avoided implementing Chaos Guardian support because it would be quite OP.

However if this is a sensible thing that a modpack creator wants to support I can look to add support for it in Woot.

**Comment by UncertifiedDevice** [comment]
Yeah it is quite OP, if you decide to support this it may be worth making it configurable but disabled by default. Thanks for taking the time to look into this by the way.

**Comment by Boomflex** [comment]
It is very OP, which is why we set it to 10 kills to achieve the item needed to spawn them.

We do custom config a lot of the mod itself to increase difficulty.

Looks like a bug though that it isn't working.

**Comment by Ipsis** [comment]
I've raised a bug in my repository to add this support.

**Comment by OrdinatorStouff** [comment]
@Ipsis I was hoping that adding the chaos dragon to the mobwhitelist and adding custom drops would have been sufficient. I agree that it might be difficult for the chaos dragon to not be quite "OP" if trying to implement a standardization inside Woot (out of the context of modpack customization).  
  
It is not my place to say what is best for your mod. I would be perfectly content with Woot having the chaos dragon blacklisted by default but enableable with the mob whitelist config, and furthermore even if whitelisted be exempted from any loot as well unless that too is configured at the pack level.  
  
In other words, adding chaos dragon support with default values might not be best for Woot mod (as you had already decided by choosing to blacklist them outright) but it would be very useful at a modpack level if we are allowed to hardcode drops and un-blacklist with configs.  
  
Whatever the final decisions, thank you in advance for looking into this and working with us.

**Comment by Ipsis** [comment]
I've got the code added to support the Chaos Dragon and the drops/capture seems to be working fine.

I'm going to enable support by default and people can disable it by adding the mob to the blacklist.
The mob was internally blacklisted as it does not die in a way that Woot can learn its drops like every other mob. So it would require custom handling inside Woot, which I currently only have for the Ender Dragon.

Since interest has been shown in a modpack wanting to allow users to farm the Chaos Dragon, I have no issue with adding the custom handling inside Woot to make this possible.

It should be available in the next release of Woot which I'm hoping to do this weekend.

**Comment by OrdinatorStouff** [comment]
@Ipsis thank you again for your support <3

**Comment by Ipsis** [comment]
Woot 1.12.2-1.0.0 has been pushed to CurseForge and should be available once they review it. This adds the Chaos Guardian support.

**Comment by OrdinatorStouff** [comment]
This should be fixed in 5.9

**Comment by DubstairNinja** [comment]
hey, just as a ordianry question.... where did you get the endershard ? can´t find a way to craft it.... is there a way to get it in survival mode ? :P sorry if it doesnt support the conversation

**Comment by Darkere** [comment]
Check JEI it should be craftable with the Stygnian Anvil.

**Comment by DubstairNinja** [comment]
> 
> 
> Check JEI it should be craftable with the Stygnian Anvil.

i thought its made out of ender pearls but its eye of ender, thanks for the quick reply, helped alot ^^
